cylc/flow/scripts/reload.py

```diff
@@ -40,6 +40,10 @@
 definitions take effect immediately, unless a task is already
 running at reload time.
 
+Upon reload, remote file installation will be triggered for all relevant
+platforms. Any changed files that are configured to be included in the file
+installation will be transferred to the appropriate remote platform.
```

Comment:
Worth making it clear this doesn't happen during the reload command itself as it did with Rose 2019 with `rose suite-run --reload`.

```suggestion
platforms on the next job submit.
Any changed files that are configured to be included in the file
installation will be transferred to the appropriate remote platform(s).
```